### ISSUE TITLE
Error creating files

### ISSUE THREAD

**Issue by padreputativo** [defold/defold#9260]
Every time I create a script, this error appears.

<hr/>

<table>
<tr><td>Build time</td><td>2024-08-03T06:49:56.452560</td></tr>
<tr><td>Defold channel</td><td>alpha</td></tr>
<tr><td>Defold editor sha</td><td>de8e81e893ced421e2a820eeeec70d78675f6212</td></tr>
<tr><td>Defold engine sha</td><td>de8e81e893ced421e2a820eeeec70d78675f6212</td></tr>
<tr><td>Defold version</td><td>1.9.2</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"7dd026abeb4b4430a18a25a72dcf762a"'>7dd026abeb4b4430a18a25a72dcf762a</a></td></tr>
<tr><td>GPU</td><td>NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2</td></tr>
<tr><td>GPU Driver</td><td>4.6.0 NVIDIA 535.183.01</td></tr>
<tr><td>Java version</td><td>17.0.5+8</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Linux</td></tr>
<tr><td>OS version</td><td>6.8.0-39-generic</td></tr>
<table>

**Comment by AGulev** [comment]
```
editor.code.data$last_visible_row.invokeStatic.class java.lang.NullPointerException: Cannot read field "dropped_line_count" because "layout" is null
    at editor.code.data$last_visible_row.invokeStatic(data.clj:1188)
    at editor.code.view$repaint_view_BANG_.invokeStatic(view.clj:3078)
    at editor.code.view$make_view_BANG_$fn.invoke(view.clj:3660)
    at clojure.lang.AFn.run(AFn.java:22)
    at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:456)
    at java.security.AccessController.doPrivileged(AccessController.java:399)
    at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:455)
    at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
    at com.sun.glass.ui.gtk.GtkApplication._runLoop(GtkApplication.java)
    at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$10(GtkApplication.java:264)
    at java.lang.Thread.run(Thread.java:833)
    ```

**Comment by britzl** [comment]
Duplicate of https://github.com/defold/defold/issues/9254

@padreputativo can you reproduce it or did it happen only once?

Which Linux distro are you using?

**Comment by padreputativo** [comment]
Sorry, I didn’t receive notifications and only found out because I was going to provide more information. I’m using a freshly installed Ubuntu to use the engine. I’ve noticed that if I press Enter to validate the information entered in a popup, it fails, but if, for example, I click the button when renaming a directory or file, it mostly works correctly unless there’s an issue due to having the collection open during that rename.

**Comment by padreputativo** [comment]
I’m currently using version 1.9.2, and right now it doesn’t happen when creating a script, whether by pressing Enter after typing the name or by clicking the Create Script button. I updated because I noticed that the other issue I reported had already been fixed. I understand that this issue might also be resolved, but in other cases, the difference between pressing Enter and using the action button does cause errors.
